update/factory/fullSyncInterceptorsContainerFactory.go

```diff
@@ -0,0 +1,470 @@
+package factory
```

Comment:
Will skip these files, they need refactoring as they won't work as expected because the network messenger does not allow adding multiple message processors on a topic